react/src/quiz/quiz.ts

```diff
@@ -81,14 +107,20 @@ export async function importQuizPersona(): Promise<void> {
     try {
         const text = await file.text()
         const persona = quizPersonaSchema.parse(JSON.parse(text))
-        if (confirm(`The uploaded progress will REPLACE ALL your Juxtastat and Retrostat progress.
+        let overwrite: boolean
+        if (localStorage.getItem('quiz_history')!==null) {  // no reason to confirm if they don't have any data in the first place'
+            overwrite = confirm(`The uploaded progress will be preferentially merged with your current Juxtastat and Retrostat progress.
 
-Your existing Juxtastat and Retrostat progress will be lost. 
+            Your existing Juxtastat and Retrostat progress, if different from what is uploaded, will be lost.
```

Comment:
```suggestion
Your existing Juxtastat and Retrostat progress, if different from what is uploaded, will be lost.
```
Otherwise the indenting will show up in the dialog